spec/swagger.yaml

```diff
@@ -124,26 +124,20 @@ tags:
-      Generally, adding multiple files to a job is associated with polling - file is parsed asynchronously and can't be added to a job until it is completed, adding file to a job is also asynchronous, and job can't be authorized until all background tasks completed.  
-      This forces third-party integration poll every file after every step.  
-
-      Batches hide these asynchronous processes from 3rd party integration - after it accepted file from client it conducts this file through all steps up to the Job authorization. As Job batch Service utilizes internal Smartling events it gets more effective than just polling files and jobs.
-      From the client side flow consists of simple steps:
-
-      1. Client creates Job (or uses existing one, which isn't currently processed with another Batch)
-      2. Client Creates a new Batch for this Job
-      3. Client adds (uploads) files into the batch giving target locales list for each file
-      4. When all files are uploaded, client triggers "execute" for the batch.
-      5. Batch execution process waiting until all files are uploaded and added to the Job and then, if was requested, authorizes the Job.
-      6. Clients can check batch status and see if all files uploaded and attached successfully, what errors happened for each file, and statistics for added/skipped strings.
-
-      After Batch is completed it can't be reused.
-
-      **Job batches Limitations**
-      - Service doesn't guarantee correct processing of multiple batches for the same job in parallel. Adding files to a new batch should be done only after previous batch reached "completed" state.
-      - Service doesn't guarantee correct processing of Batch if Job and Files are modified in UI or with another integration.
-
+      The Job Batches APIs provide a wrapper around Jobs and Files that simplifies workin with them. Users can more easily combine uploading files and adding them to Jobs.
+      Generally, adding multiple files to a Job forces the user to implement polling. This is because The Jobs API expects a file to be uploaded and fully processed before being added to a Job. In other words the Add File to Job API is *not* actually an API ~upload~, it just associates a file that was uploaded separately, to the Job. That process of uploading a file to Smartling is asynchronous.  The file needs to be fully processed before it's ready to be worked with, for example, before it's added to a Job. Lastly a Job can't be authorized until any background tasks for its component parts are completed. Job Batches simplifies these complexities.
+      The Job Batches API utilizes internal Smartling events to optimize the process of adding uploading files and adding them to Jobs.
+      The flow for the API user is as follows:
+      1. Client identifies the Job to which they want to add files. They can create a new Job or get the ID for existing one. As long as it is *not* currently being processed with another Batch)
```

Comment:
1. get the ID for existing one > get the ID for an existing one. 
2. there's a stray bracket at the end of the sentence.